src/Umbraco.Web.UI.Client/src/views/components/tree/umb-tree-item.html

```diff
@@ -9,7 +9,7 @@
             <span class="sr-only">Expand child items for {{node.name}}</span>
         </button>
 
-        <i class="icon umb-tree-icon sprTree" ng-class="node.cssClass" title="{{::node.title}}" ng-click="select(node, $event)" ng-style="::node.style" tabindex="-1" aria-hidden="true"></i>
+        <umb-icon icon="{{node.icon}}" class="icon umb-tree-icon sprTree" ng-class="node.cssClass" title="{{::node.title}}" ng-click="select(node, $event)" ng-style="::node.style" tabindex="-1" aria-hidden="true"></umb-icon>
```

Comment:
@MMasey does it still update tree icon and color in e.g. document type after changing icon via icon picker + save without reloading tree?

Also `aria-hidden` can be removed when using `<umb-icon>`.

Reply:
Good spot mate, i've now remove the aria hidden. I've also checked the functionality of the tagged PR and can confirm that it still works as intended 👍 

![umb-tree-svg-update-2](https://user-images.githubusercontent.com/6573613/95081551-9e809f00-0711-11eb-9692-4e14980ba887.gif)